src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -138,6 +138,11 @@ function toggle(a, recursive) {
     } else if (/\bsf-dump-expanded\b/.test(oldClass)) {
         arrow = '▶';
         newClass = 'sf-dump-compact';
+
+        if ('undefined' !== typeof window.Sfdump.debugBarPanel) {
```

Comment:
This means var-dumper knows something about the Symfony panel.
Can't we decouple things and alter the logic in the panel's twig file instead?

Reply:
Done. I used a custom event to decouple toggle detection and panel min-height handling.